source/StigData/Processed/WindowsServer-2016-DC-2.3.xml

```diff
@@ -3578,6 +3578,7 @@ Administrators - Full Control - This key and subkeys
 SYSTEM - Full Control - This key and subkeys
 CREATOR OWNER - Full Control - This key and subkeys
 ALL APPLICATION PACKAGES - Read - This key and subkeys
+Server Operators â€“ Read â€“ This Key and subkeys    (Domain controllers only)
```

Comment:
We need to make sure Server 2016 Domain Controller has this new User Right Assignment - Server Operators - Read - This Key and Subkeys..